### PR TITLE
Updated examples

### PR DESCRIPTION
**Pull request by kylemas**

### CODE REVIEW

File: src/examples/completeExample.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      "value": " P&D",
+      "row": 2,
+      "resolved": false
+    }
+  ],
+  "forest": [
+    {
+      "nodeType": "contradiction",
+      "formulas": [],
+      "rule": ["X"]
```
Comment (McTano) [inline src/examples/completeExample.json]: Is this supposed to be a correct example?
If so, there's a contradiction node closing the tree but no contradictory formulas, so this is incorrect.
If not, maybe change the name to make it clear this is supposed to be an incorrect example.

File: src/examples/incompleteExample.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      "value": " ~P",
+      "row": 2,
+      "resolved": false
+    }
+  ],
+  "forest": [
+    {
+      "nodeType": "finished",
+      "formulas": [],
+      "rule": ["O"]
```
Comment (McTano) [inline src/examples/incompleteExample.json]: rule should be a string, not an array.

File: src/examples/invalidExample.json [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  "forest": [
+    {
+      "nodeType": "finished",
+      "formulas": [],
+      "rule": ["O"]
+    },
+    {
+      "nodeType": "contradiction",
+      "formulas": [],
+      "rule": ["X"]
```
Comment (McTano) [inline src/examples/invalidExample.json]: See comments on previous example.

File: src/examples/completeExample.json [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  "formulas": [
+    {
+      "value": "~Q",
+      "row": 1,
+      "resolved": false
+    },
+    {
+      "value": " P&D",
+      "row": 2,
+      "resolved": false
```
Comment (McTano) [inline src/examples/completeExample.json]: For a "complete" example, all non-atomic nodes should be resolved.

**Comment by kylemas** [comment]
> It seems like you're having some trouble with this. Maybe I haven't explained clearly what we need from these examples.
> It looks like you're still creating these by hand. I think the best way to do this is to work through an example fully on the widget, and just copy the json that gets displayed below the tree. I'll do one and send you an example.
> Do some correct examples first and then we'll worry about what kind of incorrect examples we want.
> We can find a time to go over this exercise so we can figure out where the confusion is coming from.

@McTano 
Hey Tristan! Sorry about that, looks like I misunderstood what was needed. I worked through the examples and used the JSON output at the bottom of the page and added it to the files. I fixed up the correct, invalid, and complete examples but not the incorrect example.

File: src/examples/completeExample.json (hunk 4 above)
Comment (McTano) [reply src/examples/completeExample.json]: @kylemas
This formula still needs to be resolved

**Comment by McTano** [comment]
@kylemas 
This looks good now except for the unresolved formula on `completeExample`.

Please fix that and give the files more specific names. Then you can merge.

**Pull request merged**